src/Parser.php

```diff
@@ -660,6 +636,64 @@ private function parseTableRows(): array
         return $table;
     }
 
+    /**
+     * @param TToken $keywordToken
+     *
+     * @return array{title:string|null, description:string|null}
+     */
+    private function parseTitleAndDescription(array $keywordToken): array
+    {
+        $originalTitle = trim($keywordToken['value'] ?? '');
+        $textLines = [];
+
+        while (in_array($predicted = $this->predictTokenType(), ['Newline', 'Text', 'Comment'])) {
+            $token = $this->expectTokenType($predicted);
+
+            if ($token['type'] === 'Comment') {
+                continue;
+            }
+
+            $text = match ($token['type']) {
+                'Newline' => '',
+                'Text' => $token['value'],
+                default => throw new LogicException('Unexpected token type: ' . $token['type']),
```

Comment:
phpstan required this default in the match block, but it can't actually be triggered. It may be redundant once #344 is merged and phpstan has more information about the token types?

Reply:
Ah, of course. I'll leave it as a throw then.